pkg/util/utils.go

```diff
@@ -324,3 +325,67 @@ func ParseInputTime(inputTime string) (time.Time, error) {
 	}
 	return time.Now().Add(-duration), nil
 }
+
+// GetGlobalOpts checks all global flags and generates the command string
+func GetGlobalOpts(c *cliconfig.RunlabelValues) string {
```

Comment:
I wonder if we can somehow pull the names of the global flags straight out of Cobra - @baude you know if there's a way to do this?

Reply:
I found FlagSet.VisitAll() for all flags, not just for global flags...

Reply:
Hm. This could be painful if we add more global flags, but no helping that, I suppose.